src/transformState.js

```diff
@@ -5,7 +5,36 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const stateHistory = [];
```

Comment:
you weren't prompted to track state history, but it's good idea 😄

Reply:
so i keep this ;p